taxonomies/food/categories.txt

```diff
@@ -118364,3 +118393,5 @@ gpc_category_code:en: 10000613
 gpc_category_description:en: Definition: Includes any products that can be described/observed as two or more distinct Prepared and Preserved Foods sold together which exist within the schema but belong to different classes, that is two or more products contained within the same pack which cross classes within the Prepared and Preserved Foods Family. Includes products such as Chicken Korma and Ice Cream variety packs. Items that are received free with purchases should be removed from the classification decision-making process. Definition Excludes: Excludes products such as Jam and Chocolate Spread variety packs and Chips and Dips variety packs.
 gpc_category_name:en: Prepared/Preserved Foods Variety Packs
 
+
+eo: Ariane apples
```

Comment:
Eo ! @raphael0202

Reply:
I saw that! Taxonomy editor bug: https://github.com/openfoodfacts/taxonomy-editor/issues/561